mmpose/models/utils/realnvp.py

```diff
@@ -18,11 +18,12 @@ def __init__(self, get_scale_net, get_trans_net, mask, prior):
         super(RealNVP, self).__init__()
 
         self.prior = prior
-        self.register_buffer('mask', mask)
+        self.mask = mask
```

Comment:
Is there a problem with not registering the mask in the buffer? It seems the mask won't be stored in the model state dict.

Reply:
> Is there a problem with not registering the mask in the buffer? It seems the mask won't be stored in the model state dict.

The flow model won't be stored in the model state dict since I implemented RLE as a loss function.
For this line, I modified it to unify the devices of the variables of the flow model. When adding unit test and trying out, the device of `mask` will be different from`prior` using `register_buffer`.

Reply:
The device of the buffer can change with the module by simply calling `loss.cuda()`, or `loss.to(device)`, since the buffer is registered in the module and its device will stay the same as the parameters. You only need to move the device of the `prior` to GPU.

Besides, the loss module is registered in the regression head. I think the parameters of the flow model will resume with the regression head.

Reply:
You are right, I loaded the checkpoint, loss is a variable of model.head and its parameters are stored. However, it just came to my mind that should `mask` be register&stored? I remember it's a constant?
Another thing, there are some parameters that can't be personalized right now, like `prior`, `get_scale_net(nets in your code)`, `get_trans_net(nett in your code)`, and `Q`. I'm not sure whether to add options for them or not.

Reply:
Loading mask and prior to GPU are solved in commit [fix device bug of flow by overriding loss._apply(fn)](https://github.com/open-mmlab/mmpose/pull/1259/commits/76e78cd539b490e207c428deb495e3d641cc009e)